jina/parsers/peapods/pod.py

```diff
@@ -64,12 +64,9 @@ def mixin_base_pod_parser(parser):
     )
     gp.add_argument(
         '--peas-hosts',
-        action=KVAppendAction,
-        metavar='KEY: VALUE',
-        nargs='*',
-        help='''The hosts of the peas when parallel greater than 1,
-        pea have a new host address if the pea_id present in the map.
-        otherwise pea host will be identical to the host of pod.
-        Represented as a key value pair in argument.
-        key is the pea_id, and value is the host address.''',
+        nargs='+',
+        type=str,
+        help='''The hosts of the peas when parallel greater than 1.
```

Comment:
I think we need to add in the docs here that order matters, specially to reload an index

Reply:
Would you please elaborate more on why the order matters? Do you mean that the naming convention of the workspaces might cause trouble when reloading the index? Because there is Pea id information in the naming.

Reply:
Yes, that if u change the order, the pea ids will be missmatched and u will have problems when reloading the index or the executor.

Reply:
I see. I will make proper documentation for this.